src/IconButton/index.tsx

```diff
@@ -36,6 +36,8 @@ interface Props extends React.HTMLAttributes<HTMLButtonElement> {
   variant?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'light' | 'dark' | 'black' | 'brand';
   /** size of button to render */
   size?: 'sm' | 'md' | 'inline';
+  /** Used with `IconButtonToggle` */
+  value?: string;
```

Comment:
As you can see in the docs, [`<IconButtonToggle>` requires its `<IconButton>` children to accept a `value` prop](https://paragon-openedx.netlify.app/components/iconbuttontoggle/), but that wasn't defined in the prop types.

Fixed a type error:

```
Property 'value' does not exist on type 'IntrinsicAttributes & Pick<Pick<PropsWithTooltip, "variant" | "className" | "children" | "slot" | "style" | "title" | "size" | "defaultChecked" | ... 260 more ... | "tooltipContent"> & Pick<...> & Pick<...>, "tooltipContent"> & Partial<...> & Partial<...>'.
```